kalite/distributed/static/js/distributed/exercises/views.js

```diff
@@ -245,6 +240,22 @@ window.ExerciseView = Backbone.View.extend({
 
             } else if (framework == "perseus") {
 
+                var item_index;
```

Comment:
This needs to happen after data_model load, otherwise "all_assessment_items" is undefined.